front/src/common/api/osrdEditoastApi.ts

```diff
@@ -2828,6 +2927,50 @@ export type PathfindingInputV2 = {
   /** List of supported signaling systems */
   rolling_stock_supported_signaling_systems: string[];
 };
+export type ScenarioV2 = {
+  creation_date: string;
+  description: string;
+  id: number;
+  infra_id: number;
+  last_modification: string;
```

Comment:
One last question : currently on scenario v1, when we change a scenario by adding/updating/deleting a train, it doesn't update the last modification date which I think is strange. But for example, it does update the modification date on a study when we add / update / delete a scenario.

Should we add that for the v2 model ? What do you think @flomonster @leovalais ?

Reply:
We wanted to change as little as possible about the project/study/scenario part.
We can make this change later if it's the expected behavior.